runtime-2/src/runtime.rs

```diff
@@ -0,0 +1,29 @@
+use palabritas_common::Database;
+use palabritas_common::OutputText;
+use palabritas_common::Readable;
+use rand_pcg::Pcg32;
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Default, Serialize, Deserialize, PartialEq)]
+pub struct Runtime {
+  pub database: Database,
+  #[serde(skip)]
+  rng: Option<Pcg32>,
+  seed: u64,
+}
+
+impl Runtime {
+  pub fn new(database: Database) -> Runtime {
+    Runtime {
+      database,
+      ..Default::default()
+    }
+  }
+
+  pub fn next_text(&mut self) -> Option<OutputText> {
```

Comment:
why is it called `next_text` if everywhere else is called `get_next_output`? do we mean to also use the trait here?